python/demo/demo_stokes.py

```diff
@@ -74,18 +74,20 @@
 # ## Implementation
 #
 # The Stokes problem using Taylor-Hood elements is solved using:
-# 1. [Block preconditioner using PETSc MatNest and VecNest data
-#    structures. Each 'block' is a standalone object.]
-#    (#nested-matrix-solver)
+# 1. [Block preconditioner using PETSc Nest data structures using
```

Comment:
"using PETSc Nest data structures using the high-level interface" reads strangely.

Also not clear what the "high-level interface" is.

Reply:
Do you have a better suggestion? Currently the link is completely broken
<img width="757" height="167" alt="image" src="https://github.com/user-attachments/assets/c19c9b1e-21c0-4061-9d9a-4ab07011d57f" />